contracts/Multi-signature/MultiplexingContract.sol

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright: Ambrosus Inc.
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "../Front/KycWhitelist.sol";
+import "../Consensus/BlockRewards.sol";
+import "../Consensus/ValidatorSet.sol";
+
+
+contract MultiplexingContract is Ownable {
+
+    address[] private _controlledContracts;
+    mapping (address => uint) private _indexes;
+
+    function performTransaction(address executor, bytes memory transaction) public {
+        bool res = executor.call(transaction);
```

Comment:
For now please just remove this file and its tests and include this line in the ApprovalCollector contract. Let's focus on getting the multisig contract right, and then we'll go to implementing multiplexing.

Reply:
I wasn't able to remove file, because I need real contract interface for creating stub . I'll create stub and update tests right after real multiplexing contract implementation. For now I only removed all logic from contract.